src/hitbox.js

```diff
@@ -1,6 +1,4 @@
-import Chart from 'chart.js';
-
-var helpers = Chart.helpers;
+import {merge} from 'chart.js/helpers';
```

Comment:
Travis complains about this line: 

> 'chart.js/helpers' is imported by src/hitbox.js, but could not be resolved – treating it as an external dependency. No name was provided for external module 'chart.js/helpers' in output.globals – guessing 'helpers$4'

 I'm not sure this can work with this rollup config. Let's keep it simple and use the previous code (i.e. `helpers.merge`).